src/common/helpers.ts

```diff
@@ -265,6 +266,56 @@ const isIpAddress = (name: string): boolean => {
   return false;
 };
 
+const getLastUpdated = (credential: CredentialType): string => {
```

Comment:
Is there an unit test for this? If not, do you think worth it have one?

Reply:
yes, I def want to cover these in tests.  I am just going step by step. =)